server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsResponse.java

```diff
@@ -105,6 +107,36 @@ public ClusterStatsResponse(
         this.status = status;
     }
 
+    public ClusterStatsResponse(
```

Comment:
I think with a little refactoring you should be able to invoke this constructor from the above constructor at L67 and reduce some code duplicacy.

Reply:
Done.